src/components/search/Search.js

```diff
@@ -4,20 +4,21 @@ import {LocationInput} from './LocationInput';
 
 export const Search = (props) => {
 	return (
-	<div>
-	<div className='location-area'>
-	    <LocationInput
-	      value = {props.name}
-	      handleLocationInput={props.handleLocationInput}
-	      handleLocationSelect={props.handleLocationSelect}
-	    />
-	  </div>
-	  <DateInput
-	    handleDateInput={props.handleDateInput}
-	    date={props.date}
-	  />
-	  <p style={{float: 'left', marginTop: '1rem'}}>
-	    {props.length || "No"} events found
-	  </p>
-	</div>)
+		<div>
```

Comment:
The indentation looks off in this view for some reason. With tab space set to 2 it looks like [this](https://github.com/hire-notes/react-event-search/blob/37a3c27fcf84e76d94bcfef10b5df6913a7bbfe5/src/components/search/Search.js?ts=2)